core/model/commands/transfer.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+http://soramitsu.co.jp
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef CORE_DOMAIN_TRANSACTIONS_TRANSFERTRANSACTION_HPP_
+#define CORE_DOMAIN_TRANSACTIONS_TRANSFERTRANSACTION_HPP_
+
+#include "transaction.hpp"
+#include "command.hpp"
+#include "objects.hpp"
+
+#include <string>
+#include <type_traits>
+
+#include "../../service/json_parse.hpp"
+
+namespace command {
+
+    template<typename T,
+        std::enable_if_t<
+            std::is_base_of<Objects, T>::value,std::nullptr_t
+        > = nullptr
+    >
+    class Transfer : public Command{
+        T object;
+    public:
+        Transfer(T object):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class 'Transfer' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/file/4345543347/issues/source?bid=3712491&fileBranchId=3870389#l40)